dephell_pythons/_shell_utils.py

```diff
@@ -13,7 +13,7 @@ def is_executable(path: Path) -> bool:
     try:
         if not path.is_file():
             return False
-    except PermissionError:
+    except (PermissionError, OSError):
```

Comment:
`PermissionError` is a subclass of `OSError`

Reply:
Fixed